dataland-frontend/package.json

```diff
@@ -35,27 +35,27 @@
   },
   "devDependencies": {
     "@cypress/code-coverage": "^3.10.0",
-    "@faker-js/faker": "^7.5.0",
+    "@faker-js/faker": "^7.6.0",
     "@types/big.js": "^6.1.6",
-    "@types/node": "^18.7.18",
-    "@vitejs/plugin-vue": "^3.1.0",
-    "@vue/eslint-config-typescript": "^11.0.1",
-    "@vue/test-utils": "^2.0.2",
+    "@types/node": "^18.11.0",
+    "@vitejs/plugin-vue": "^3.1.2",
+    "@vue/eslint-config-typescript": "^11.0.2",
+    "@vue/test-utils": "^2.1.0",
     "@vue/tsconfig": "^0.1.3",
     "big.js": "^6.2.1",
-    "cypress": "^10.8.0",
+    "cypress": "^10.10.0",
```

Comment:
In einer anderen Änderung switchst du das auf 10.9.0.  Wieso hierauf 10.10.0?

Reply:
Gleicher Grund wie unten. 10.10.0 ist als dev getaggt, 10.9.0 als stable. Würde hier dann auch auf 10.9.0 gehen

Reply:
Ich hatte keine Probleme beobachtet, plus die Version ist auch auf der cypress Seite als aktuelle Version gelistet ohne Kennzeichnung als experimentell.